lib/ui/form/widget/form_survey_answer_multi_choice.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/model/question_model.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+class FormSurveyAnswerMultiChoice extends StatefulWidget {
+  final QuestionModel question;
+
+  const FormSurveyAnswerMultiChoice({super.key, required this.question});
+
+  @override
+  State<FormSurveyAnswerMultiChoice> createState() =>
+      _FormSurveyAnswerMultiChoiceState();
+}
+
+class _FormSurveyAnswerMultiChoiceState
+    extends State<FormSurveyAnswerMultiChoice> {
+  final List<int> _selectedIndexes = [];
+  late List<String> _options = [];
+
+  @override
+  void initState() {
+    super.initState();
+  }
```

Comment:
Same here

Reply:
As options are fixed I read the options here now, updated the code